source/_components/satel_integra.markdown

```diff
@@ -59,16 +61,20 @@ port:
   required: false
   default: 7094
   type: integer
-partition:
-  description: The partition to operate on. Integra can support multiple partitions, this platform only supports one.
-  required: false
-  default: 1
-  type: integer
-arm_home_mode:
-  description: The mode in which arm Satel Integra when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what are the differences between them, please refer to Satel Integra manual.
-  required: false
-  default: 1
-  type: integer
+partitions:
+  description: List of the partitions to operate on.
+  required: true
```

Comment:
This is optional according to the source.

Reply:
The default is an empty dict.

Reply:
Indeed, it's not necessary, one can decide to use just inputs/outputs of the alarm without mapping its state into HA.

Reply:
Fixed.